app/models/summerExperience.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.models import *
+from app.models.user import User
+
+class SummerExperience(baseModel):
+    user = ForeignKeyField(User)
+    studentName = CharField()
+    summerYear = CharField()
+    roleDescription = TextField()
+    experienceType = CharField()
+    CceMinorContentArea = TextField()  # Store as comma-separated values or use a related table if needed
```

Comment:
This name is weird, because all summer experiences are part of the CCE Minor (also don't capitalize variables, and it should be plural). let's just go with `contentAreas`

Reply:
fixed